Sources/TuistCache/ContentHashing/TargetScriptsContentHasher.swift

```diff
@@ -27,10 +27,14 @@ public final class TargetScriptsContentHasher: TargetScriptsContentHashing {
         for script in targetScripts {
             var pathsToHash: [AbsolutePath] = []
             script.path.map { pathsToHash.append($0) }
-            pathsToHash.append(contentsOf: script.inputPaths)
-            pathsToHash.append(contentsOf: script.inputFileListPaths)
-            pathsToHash.append(contentsOf: script.outputPaths)
-            pathsToHash.append(contentsOf: script.outputFileListPaths)
+            let scriptPaths = script.inputPaths + script.inputFileListPaths + script.outputPaths + script.outputFileListPaths
```

Comment:
Because paths with variables can only be resolved at build time, the hashing of the string won't hash the content of the file the string points to, and that might lead to non-deterministic caching behaviors. What do you think if we warn the user about this so that they are aware of it?

Reply:
yes, reasonable. I'll be adding the warning on those lines

Reply:
@pepicrft I think it's done, please have a look